### ISSUE TITLE
[Developer] [Module 7] Displayed vs hidden

### ISSUE THREAD

**Issue by hidde** [w3c/wai-curricula#298]
Regarding: 

> Students are presented with different application views and are asked to code their displayed or hidden states using aria-hidden together with its corresponding value. Assess students’ knowledge of how to code displayed and hidden states of application views.

Having ‘displayed’ and ‘hidden’ written as code confused me a bit, because it reminded me of the `display` property in CSS and `hidden` attribute in HTML.

Maybe somewhere we should also mention hiding with the `hidden`-attribute/`display:none` and how it is different from the `aria-hidden` attribute.

**Comment by nitedog** [comment]
How about this:

> Students are presented with an application with content that can be shown to or hidden from the user; for example, an application to create presentation slides. Students are asked to ensure that the code reflects the shown and hidden states of the content; for example, by using the WAI-ARIA attribute `aria-hidden`, the HTML attribute `hidden`, and the CSS property `display: none`.

**Comment by hidde** [comment]
Sounds good. That doesn't have all the nuance (that `aria-hidden` hides from assistive technologies, but leaves content visible on the page, while `hidden`/`display:none` hide visually _and_ from assistive technologies), but that may be fine in this context.

**Comment by daniel-montalvo** [comment]
I suggest:

> * Practical &mdash; Students are presented with an application with content that can be shown to or hidden from the user; for example, an application to create presentation slides. Students are asked to ensure that the code reflects the shown and hidden states of the content; for example, by using the WAI-ARIA attribute `aria-hidden`, the HTML attribute `hidden`, and the CSS property `display: none`. Assess students' knowledge of how to code `displayed` and `hidden` states of application views.

@hidde , After these clarifications, my understanding is that the markup of the attributes and properties no longer distracts you. Correct me if I'm wrong.

We use this markup through the whole resource when we point to specific elements, attributes, or properties. It would be weird to use it in other places and not here.
